packages/firestore/test/integration/api/server_timestamp.test.ts

```diff
@@ -245,11 +245,12 @@ apiDescribe('Server Timestamps', persistence => {
   it('can return previous value through consecutive updates', () => {
     return withTestSetup(() => {
       return writeInitialData()
-        .then(() => docRef.firestore.disableNetwork)
+        .then(() => docRef.firestore.disableNetwork())
         .then(() => {
           // We set up two consecutive writes with server timestamps.
           docRef.update('a', FieldValue.serverTimestamp());
-          docRef.update('a', FieldValue.serverTimestamp());
+          // include b=1 to ensure there's a change resulting in a new snapshot.
+          docRef.update('a', FieldValue.serverTimestamp(), 'b', 1);
```

Comment:
This is the actual fix. The rest is just making the test actually disable/re-enable the network and match the other ports.